src/app/api/v1/webhooks/stripe/connect/route.ts

```diff
@@ -0,0 +1,61 @@
+import to from 'await-to-js'
+import { headers } from 'next/headers'
+import { NextResponse } from 'next/server'
+
+import { stripe } from '@/server/services/stripe'
+import { logger } from '@/server/lib/logger'
+import { createAdminClient } from '@/utils/supabase/serverAdmin'
+
+const endpointSecret = process.env.STRIPE_CONNECT_WEBHOOK_SECRET
+
+if (!endpointSecret)
+  throw new Error('STRIPE_CONNECT_WEBHOOK_SECRET env var is not set')
+
+export const POST = async (req: Request) => {
+  const body = await req.text()
+  const signature = headers().get('Stripe-Signature') as string
+
+  const [err, event] = await to(
+    stripe.webhooks.constructEventAsync(body, signature, endpointSecret),
+  )
```

Comment:
This webhook handler is actually pretty noisy. I believe I received 10 or so events whilst I was completing Stripe onboarding as a service-provider

Reply:
It isn't an issue now since we don't have any users, but this could revisit this later with a Webhook API server.